### ISSUE TITLE
Skills Picker- Keyboard / Focus Trap 

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#4761]
## Description
When the user tries to navigate the skills picker (or any scrollable element) they are confronted with a keyboard trap that keeps them in the element.  Any element should be navigable by keyboard using the following keys: 
1.  Tab into component,
2. Use arrow keys to navigate through links↑/↓ or ←/→ - increase or decrease slider value, Home/End - beginning or end, 
3. **Tab out of component**

This fix will be important for Fable Testing in the coming weeks :).

## Screenshot: Production
[Focus Trap issue - Skills picker](https://user-images.githubusercontent.com/101275838/202018126-3d926d0b-2915-4646-9c72-746cb9d1ca16.PNG)

**Comment by esizer** [comment]
I think the fix for this would be a skip link. I'm not sure we have the ability to tab into this since it is just using default browser behaviour when it comes to scrolling.

The trap that is happening is that the element contains, by default, hundreds of links which are focusable.

**Comment by esizer** [comment]
@MaddyDaigle Is the rework of this component in the design pipeline? Seems like it could still use some work 😓

**Comment by MaddyDaigle** [comment]
@gobyrne @Jerryescandon I definitely think is should be.  I think the dropdown part is good, in terms of the list... would you like me to take another crack at it?  I have a few ideas that I'd like to try out to improve.

**Comment by esizer** [comment]
Speaking with @RM-1978 we are going to use a temporary fix of `ctrl`+`shift`+`esc` to jump past all of the "add skill" buttons for now while the design team look for a more permanent fix.

This should only occur while inside the scrollable area.

```tsx
const skillListRef = React.useRef<HTMLDivElement>(null);

React.useEffect(() => {
  const handler = (event) => {};
  if(skillListRef.current) {
    const skillEscape = skillListRef.current.addEventListener('keydown', handler);
    
    return = () => skillListRef.current.removeEventListener('keydown', handler);
  }
}, [skillListRef]);
```

**Comment by MaddyDaigle** [comment]
Related to issue #4794
